api/lanes.py

```diff
@@ -562,3 +590,59 @@ def apply_filters(self, qu, work_model=Work, *args, **kwargs):
         qu = qu.join(work_edition).filter(work_edition.series==self.series)
         qu = qu.order_by(work_edition.series_position, work_edition.title)
         return qu
+
+
+class ContributorLane(QueryGeneratedLane):
+    """A lane of Works written by a particular contributor"""
+
+    ROUTE = 'contributor'
+    MAX_CACHE_AGE = 48*60*60    # 48 hours
+
+    def __init__(self, _db, contributor_name, contributor_id=None):
```

Comment:
At this moment, there's no point when the URL has the contributor_id; it's only passed in when the RelatedBooksLane is initialized. I think I may have been thinking about the connection between route and lane too strictly. I can remove contributor_id from the route, but leave it here. Either way, the cached feed will have used the ID, but it won't be present or shared anywhere in the feed; the URL that the lane generates uses only the contributor display_name.